dtfs-central-api/src/v1/routes/middleware/payload-validation/schemas/tfm-team.schema.ts

```diff
@@ -0,0 +1,4 @@
+import { TEAM_IDS, TeamId } from '@ukef/dtfs2-common';
+import z from 'zod';
+
+export const TfmTeamSchema = z.enum(Object.values(TEAM_IDS) as [TeamId, ...TeamId[]]);
```

Comment:
This is because of how `zod` works - when you create a `zod` `enum`, it expects an array with at least one element (the type is `[string, ...string[]]`. `Object.values(TEAM_IDS)` returns `('PDC_RECONCILE' | 'PDC_READ' | ...)[]` which could technically be an empty array, therefore throwing a type error